app/views/course/settings/components.html.slim

```diff
@@ -0,0 +1,15 @@
+h2 = t('.header')
+= simple_form_for @settings, url: course_components_path, method: :patch do |f|
+  table.table
+    thead
+      th = t('.name')
+      th = t('.enabled')
+    tbody
+      - instance_components = controller.current_component_host.instance_components
```

Comment:
No, you said don't need to add a `helper_method ` to  `current_component_host ` before :)